### PR TITLE
Longhaul/Stress: Eliminate pipeline parameter logType and logAnalyticsEnabled

### PR DESCRIPTION
**Pull request by and-rewsmith**
One log type is not sufficient to handle the different report structures that the analyzer will upload (messages, twins, direct methods). I am eliminating this parameter, while also eliminating the configuration for log analytics to be enabled / disabled. It should always be enabled.

### CODE REVIEW

**Comment by and-rewsmith** [comment]
I am in the process of building the images and testing longhaul / stress

File: scripts/linux/runE2ETest.sh [hunk 1]
```diff
@@ -447,19 +439,17 @@ function process_args() {
...
+                '-twinUpdateSize' ) saveNextArg=34;;
+                '-twinUpdateFrequency' ) saveNextArg=35;;
+                '-twinUpdateFailureThreshold' ) saveNextArg=36;;
+                '-metricsEndpointsCSV' ) saveNextArg=37;;
+                '-metricsScrapeFrequencyInSecs' ) saveNextArg=38;;
+                '-metricsUploadTarget' ) saveNextArg=39;;
+                '-hostPlatform' ) saveNextArg=40;;
+                '-initializeWithAgentArtifact' ) saveNextArg=41;;
+                '-testInfo' ) saveNextArg=42;;
                 '-cleanAll' ) CLEAN_ALL=1;;
```
Comment (philipktlin) [inline scripts/linux/runE2ETest.sh]: Clean up line 477-481; you may not be able to remove log analytics enabled flag, as it is needed to set false for e2e tests.

File: scripts/linux/runE2ETest.sh [hunk 2]
```diff
@@ -1015,6 +1000,17 @@ function validate_test_parameters() {
...
+        if [[ -z "$LOG_ANALYTICS_WORKSPACE_ID" ]]; then
+            print_error "Required log analytics workspace id"
+            ((error++))
+        fi
+
+        if [[ -z "$LOG_ANALYTICS_SHARED_KEY" ]]; then
+            print_error "Required log analytics shared key"
+            ((error++))
+        fi
+
```
Comment (philipktlin) [inline scripts/linux/runE2ETest.sh]: nit: remove blank line

Comment (and-rewsmith) [reply scripts/linux/runE2ETest.sh]: good catch

**Pull request merged**